webpack/assets/javascripts/react_app/components/notifications/drawer/notificationGroup.js

```diff
@@ -39,9 +40,17 @@ export default (
               onClickedLink={onClickedLink}
               key={notification.id}
               notification={notification}
-              onMarkAsRead={onMarkAsRead.bind(this, group)}
+              onMarkAsRead={onMarkAsRead.bind(this, group, [notification.id])}
```

Comment:
first, it's a different API call, also reducer handles it differently, and also it is called from a different place, with different parameters.
so i'm not sure there is any duplication here :)